### PR TITLE
Added support for ssl tests in docker script

### PR DESCRIPTION
**Pull request by ShivanshGahlot**
PG, Mysql ssl tests work fine.
Oracle ssl tests work with debezium only right now
Will also do a little restructuring in the docker script.

### CODE REVIEW

File: docker/yb-voyager-docker [hunk 1]
```diff
@@ -95,6 +95,95 @@ do
...
+		source_ssl_cert_file=$(basename "$source_ssl_cert")
+		if [[ $OSTYPE == 'darwin'* ]]
+		then
+			docker volume rm voyager-source-ssl-cert-dir
+			docker volume create --driver local --opt type=none --opt device="${source_ssl_cert_dir}" --opt o=bind voyager-source-ssl-cert-dir
+			source_ssl_cert_vol="-v voyager-source-ssl-cert-dir:/source-ssl-cert-dir"
+		else
+			source_ssl_cert_vol="-v ${source_ssl_cert_dir}:/source-ssl-cert-dir"
+		fi
+		argv[$j]='/source-ssl-cert-dir/'$source_ssl_cert_file
```
Comment (amit-yb) [inline docker/yb-voyager-docker:114]: See if you can factor out this in a separate function named `map_path_inside_container(path, path_in_container)`

File: docker/yb-voyager-docker [hunk 2]
```diff
@@ -79,22 +117,27 @@ do
...
 
+	# If the flag is --oracle-tns-alias then find the exported variable TNS_ADMIN and map it to the container.  --oracle-tns-alias flag will be passed as it is.
+	if [[ ${argv[${i}]} == "--oracle-tns-alias" ]]
+	then
+		tns_admin=$(env | grep -E '^TNS_ADMIN')	
+		if [[ -n "$tns_admin" ]]; then
+			tns_admin=$(echo $tns_admin | cut -d '=' -f2)
+		else 
+			tns_admin="${ORACLE_HOME}/network/admin"
+		fi
```
Comment (amit-yb) [inline docker/yb-voyager-docker:133]: How about the following approach:

`tns_admin=${TNS_ADMIN:"${ORACLE_HOME}/network/admin"}`

File: docker/yb-voyager-docker [hunk 3]
```diff
@@ -27,29 +45,49 @@ for var_name in "${variables[@]}"; do
...
+exported_vars="${exported_vars} -e LC_ALL=en_US.UTF-8 -e LC_CTYPE=en_US.UTF-8 -e ORACLE_HOME=/usr/lib/oracle/21/client64 -e LD_LIBRARY_PATH=/usr/lib/oracle/21/client64/lib -e PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/local/go/bin:/usr/lib/oracle/21/client64/bin"
 
+map_host_path_inside_container() {
+		flag_name=$1
+		dir=$2
+		# If flag_name is "" then we have to extract it from argv
+		if [[ $flag_name == "" ]]
+		then
+			flag_name=argv[${i}]
+			j=$(( $i + 1))
```
Comment (amit-yb) [inline docker/yb-voyager-docker:57]: Is this required anymore?

File: docker/yb-voyager-docker [hunk 4]
```diff
@@ -27,29 +45,49 @@ for var_name in "${variables[@]}"; do
...
-	if [[ ${argv[${i}]} == "--export-dir" || ${argv[${i}]} == "-e" ]]
-	then
-		j=$(( $i + 1))
-		export_dir=${argv[${j}]}
-		export_dir=$(realpath "$export_dir")
+exported_vars="${exported_vars} -e LC_ALL=en_US.UTF-8 -e LC_CTYPE=en_US.UTF-8 -e ORACLE_HOME=/usr/lib/oracle/21/client64 -e LD_LIBRARY_PATH=/usr/lib/oracle/21/client64/lib -e PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/local/go/bin:/usr/lib/oracle/21/client64/bin"
 
+map_host_path_inside_container() {
+		flag_name=$1
+		dir=$2
```
Comment (amit-yb) [inline docker/yb-voyager-docker:52]: Looks like `dir` is a name of an existing command from coreutils. Consider changing this name to `dir_path`.

File: docker/yb-voyager-docker (hunk 2 above)
Comment (amit-yb) [inline docker/yb-voyager-docker:133]: `tns_admin=${TNS_ADMIN:"${ORACLE_HOME}/network/admin"}`

**Review by amit-yb — APPROVED** [review]
LGTM

File: docker/yb-voyager-docker [hunk 5]
```diff
@@ -15,9 +17,25 @@ fi
...
 i=0
 exported_vars=""
+volume_mappings=""
 
 # Define an array of environment variable names to capture
-variables=("BETA_FAST_DATA_EXPORT" "SOURCE_DB_PASSWORD" "TARGET_DB_PASSWORD" "FF_DB_PASSWORD" "YB_MASTER_PORT" "YB_TSERVER_PORT" "QUEUE_SEGMENT_MAX_BYTES" "NUM_EVENT_CHANNELS" "EVENT_CHANNEL_SIZE" "MAX_EVENTS_PER_BATCH" "MAX_INTERVAL_BETWEEN_BATCHES")
+variables=("BETA_FAST_DATA_EXPORT" 
+"SOURCE_DB_PASSWORD" 
+"TARGET_DB_PASSWORD" 
+"FF_DB_PASSWORD"
```
Comment (priyanshi-yb) [inline docker/yb-voyager-docker:26]: @ShivanshGahlot  Change FF_DB_PASSWORD -> SOURCE_REPLICA_DB_PASSWORD 
and add
`YB_VOYAGER_SEND_DIAGNOSTICS`

**Pull request merged**